### PR TITLE
Add QE UPI zonal workflow

### PR DESCRIPTION
**Pull request by sgaoshang**

### CODE REVIEW

**Comment by sgaoshang** [comment]
/pj-rehearse periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-vsphere-upi-zones

File: ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/zones/provision/cucushift-installer-rehearse-vsphere-upi-zones-provision-chain.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-upi-zones-provision
+  steps:
+  - chain: upi-vsphere-pre-zones
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/zones/provision/cucushift-installer-rehearse-vsphere-upi-zones-provision-chain.yaml:4]: Suggest to flatten the chain to:
```
  - ref: ipi-install-rbac
  - ref: openshift-cluster-bot-rbac
  - chain: upi-conf-vsphere-zones
  - ref: upi-install-vsphere
  - ref: ipi-install-times-collection
```
To easily read its real steps.

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/vsphere/upi/zones/provision/cucushift-installer-rehearse-vsphere-upi-zones-provision-chain.yaml:4]: Also missing `  - ref: enable-qe-catalogsource` step

**Comment by jianlinliu** [comment]
@sgaoshang pls remember to add ci jobs latter for it.

**Comment by sgaoshang** [comment]
> @sgaoshang pls remember to add ci jobs latter for it.

Sure, will add ci job when this PR merged

**Pull request merged**